### DISCUSSION TITLE
How to write tag inside `<head>` in 'use client'

### DISCUSSION THREAD

**Discussion by hyeongrok7874** [vercel/next.js#52574] [Help] [answered]
### Summary

In my project, I am using emotion to style.

However, while using emotion, I know that there is a problem that 'use client' must be used for all pages.
(https://github.com/emotion-js/emotion/issues/2928)

Of course, we are aware that the use of emotion is currently discouraged.

In the current state, metadata api and generateMetadata do not work, and next/head cannot be used either.

I want to write tags in the head, such as meta tags, in the page, but I can't find a way.

Any help would be appreciated 🙇‍♂️

**Comment by icyJoseph** [comment, marked as answer]
Refactor.

```tsx
'use client'

export default function Page() {
   return /* code that uses emotion API */
}
```

Into a Server Component at the page level, that just returns a ClientComponent as JSX.

```tsx
import { HomePageWithEmotionStyles } from './HomePage';

export const metadata = { /* stuff goes here */ }

export default function Page() {
   return <HomePageWithEmotionStyles />
 }
```

**Reply by hyeongrok7874** [reply]
In the end, thanks to this, I was able to see that the use of server components was inevitable for pages.


In my case, I also had to remove `compiler: {emotion: true}` in `next config` and `{jsxImportSource: '@emotion/react'}` in `tsconfig`.

I hope this helps others who have the same problem.

Also, I wish I had known earlier that it is still difficult to apply emotions to app routers.

Thank you very much for your reply.
